### PR TITLE
ci(HMS-3962): Update Integration test image for IQE

### PR DESCRIPTION
**Pull request by akhil-jha**
Name update from hms-integration to insights-experiences.

### CODE REVIEW

**Comment by akhil-jha** [comment]
@jrusz

**Comment by ezr-ondrej** [comment]
@akhil-jha the kube-linter is unhappy. Do you think we can fix that, or shall we adjust the linter?

```
The container "image-builder-iqe--tests-image_tag-v0hqck" is using an invalid container image, "quay.io/cloudservices/iqe-tests". Please use images that are not blocked by the `BlockList` criteria : [".*:(latest)$" "^[^:]*$" "(.*/[^:]+)$"] (check: latest-tag, remediation: Use a container image with a specific tag other than latest.)
```

File: templates/iqe-trigger-integration.yml [hunk 1]
```diff
@@ -124,13 +124,16 @@ parameters:
   from: "[a-z0-9]{6}"
 - name: IQE_IMAGE
   description: "container image path for the iqe plugin"
-  value: quay.io/cloudservices/iqe-tests:hms-integration
+  value: quay.io/cloudservices/iqe-tests
```
Comment (jrusz) [inline templates/iqe-trigger-integration.yml:127]:
```suggestion
  value: quay.io/cloudservices/iqe-tests:insights-experiences
```
I guess let's add this here, that should make the linter happy and it makes sense to me to have a tag there.

**Review by jrusz — CHANGES_REQUESTED** [review]

**Review by jrusz — APPROVED** [review]

**Pull request merged**